### PR TITLE
Add Console release notes

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-03-13-console-release-notes-neondatabase.vercel.app/docs/release-notes/2023-03-14

Note: Now includes mention of project sharing and Pro plan related changes. Since we'll publish & release it all today, we can mention these items here.

### CODE REVIEW

File: content/release-notes/2023-03-10.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+label: 'Console'
```
Comment (Daniel-ef) [inline content/release-notes/2023-03-10.md:2]: Actually, the Release wasn't rolled out due to a bug in Control Plan. 
So I would put everything into the `2023-03-14` notes except "Reduced the number of queries performed". It was reverted and has not merged with fix yet.

Comment (danieltprice) [reply content/release-notes/2023-03-10.md]: Thank you. Moved all notes in the 03-14 notes. Dropped the reverted item.

**Review by Daniel-ef — APPROVED** [review]

File: content/release-notes/2023-03-14.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- API: Removed the `physical_size` property from the Branch response body in the Neon API.
+- UI: Changed the **User** label on the **Connection Details** widget to **Roles**. Database users are now referred to as **Roles** in Neon.
+- UI: Added descriptive text and a documentation link to the **Integrations** page in the Neon Console.
+- UI: Added support for sharing projects with other Neon users. Project sharing is performed from the **Settings** page in the Neon Console. The project sharing feature is available only to Neon paid plan users, but projects can be shared with any Neon user, including Free Tier users. For instructions, see [Manage projects](/docs/manage/projects).
+- UI: Added an **Upgrade to Pro** option to the Neon Console to enable upgrading from Free Tier to Neon's Pro plan. For information about Neon's paid plans, please refer to [Billing](/docs/introduction/billing).
+
+### Bug Fixes
+
+- Control Plane: Addressed an issue with the Activity Monitor that caused unexpected CPU usage. The Activity Monitor is a process that monitors Neon compute endpoints for activity.
```
Comment (Daniel-ef) [inline content/release-notes/2023-03-14.md:15]: This can be deleted too

Comment (danieltprice) [reply content/release-notes/2023-03-14.md:15]: Thank you. Removed.

**Review by Daniel-ef — APPROVED** [review]

**Pull request merged**